test/testzsync.cpp

```diff
@@ -42,12 +42,17 @@ private slots:
         SyncOptions opt;
         opt._deltaSyncEnabled = true;
         opt._deltaSyncMinFileSize = 0;
+        // Reduce the chunk size, allowing us to work with smaller data
+        int chunkSize = 2 * 1000 * 1001;
```

Comment:
Note: since we build with C++14,  we can also use the group separator notation: `2'002'000`  It might actually be more readable.

Reply:
might also be nice in future to name variables with a unit.
E.g. chunkSizeBytes, timeOutMsec